randovania/games/planets_zebeth/assets/hash_words.json

```diff
@@ -0,0 +1,65 @@
+[
+    "Acid",
+    "Atomic",
+    "Attack",
+    "Ball",
+    "Beam",
+    "Blue",
+    "Bomb",
+    "Boots",
+    "Brain",
+    "Brick",
+    "Bridge",
+    "Brinstar",
+    "Bubble",
+    "Dessgeega",
+    "Energy",
+    "Gamet",
+    "Geruta",
+    "Hi-Jump",
+    "Hive",
```

Comment:
Metroid 1 doesn't feature Hive's (assuming that this is referencing the hives for mellows).